docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md

```diff
@@ -0,0 +1,333 @@
+---
+title: "Manage SOA domains"
+date: 2019-04-18T06:46:23-05:00
+weight: 2
+description: "SOA domains include the deployment of various Oracle Service-Oriented Architecture (SOA) Suite components, such as SOA, Oracle Service Bus (OSB), and Oracle Enterprise Scheduler (ESS)."
+---
+
+#### Contents
+
+* [Limitations](#limitations)
```

Comment:
Please start off this list with "Introduction"
[Introduction](#introduction)

Reply:
The bulleted list is a Table of Contents for the doc, hence the title "Contents." If you want me to place the heading, "Introduction," after the TOC and preceding the first paragraph, I will do that. Is that ok?

Reply:
I meant the first point in the bullet list is "Introduction". Then the next point is "Limitations".

Reply:
Got it.